scripts/grounding/assembly.sbt

```diff
@@ -0,0 +1,30 @@
+assembly / aggregate := false
+assembly / assemblyMergeStrategy := {
+/*
+  // See https://github.com/sbt/sbt-assembly.
```

Comment:
Do we need to keep this commented-out code?

Reply:
Noe